wrapanapi/systems/ec2.py

```diff
@@ -1421,3 +1429,26 @@ def list_regions(self, verbose=False):
             ssm_response = self.ssm_connection.get_parameter(Name=tmp)
             verbose_region_names.append(ssm_response['Parameter']['Value'])
         return verbose_region_names
+
+    def create_stack(self, name, template_url=None, template_body=None, parameters=None,
+                     capabilities=None):
+        if not template_body and not template_url or template_body and template_url:
+            raise ValueError("Either template_body or template_url must be set and not both!")
+        stack_kwargs = {
+            'StackName': name,
+        }
+        if template_body:
+            stack_kwargs['TemplateBody'] = template_body
+        else:
+            stack_kwargs['TemplateURL'] = template_url
+        if parameters:
+            stack_kwargs['Parameters'] = parameters
+        if capabilities:
+            stack_kwargs['Capabilities'] = capabilities
+        try:
+            response = self.cloudformation_connection.create_stack(**stack_kwargs)
+            stack_id = response.get('StackId')
+            return CloudFormationStack(system=self, uuid=stack_id,
+                                       raw=self.cloudformation_resource.Stack(stack_id))
+        except Exception:
```

Comment:
**Question**: Is there a more specific exception that you can catch?

Reply:
I'm more concerned with why the try/except is here? Its not handling the exception at all, might as well just let the exception be raised by any of the calls in the try, without catching them here.

Reply:
I deleted try/except.
For me it's much better that I get exception instead True/False in Integration tests so I know what's failing in wrapanapi. When I am fixing tests I have to remove try/except to debug the issue.